docs/source/conf.py

```diff
@@ -94,6 +95,12 @@
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = False
 
+# Enable linking references to other projects' documentation.
+intersphinx_mapping = {
+    'python': ('https://docs.python.org/3', None),
+    'matplotlib': ('https://matplotlib.org/stable', None),
+}
+
```

Comment:
where is this being used?

Reply:
In the API docs, `pynucastro.networks.base_cxx_network.BaseCxxNetwork` references `abc.ABC` and `pynucastro.rates.rate.{ReacLibRate,TabularRate}.plot` both reference `matplotlib.figure.Figure`.

Reply:
ah okay